groups/sig-node/groups.yaml

```diff
@@ -32,6 +32,29 @@ groups:
       - mpatel@redhat.com
       - s.kanzhelev@gmail.com
 
+  - email-id: sig-node@kubernetes.io
+    name: sig-node
+    description: |-
+      SIG node general discussion group
+    owners:
+      - killen.bob@gmail.com # temporary to finish configuration
+      - dawnchen@google.com
+      - decarr@redhat.com
+      - mpatel@redhat.com
+      - s.kanzhelev@gmail.com
+    settings:
+      WhoCanJoin: "ANYONE_CAN_JOIN"
+      WhoCanViewGroup: "ANYONE_CAN_VIEW"
+      WhoCanDiscoverGroup: "ANYONE_CAN_DISCOVER"
+      WhoCanPostMessage: "ANYONE_CAN_POST"
+      MessageModerationLevel: "MODERATE_NON_MEMBERS"
+      WhoCanViewMembership: "ALL_MANAGERS_CAN_VIEW"
```

Comment:
Is there any issues with exposing the membership to members? In WG serving we found that without this setting members cannot invite the whole group to the meeting. Is there some privacy concerns to open membership view to everyone?

Reply:
Not really, it was a default from before. Good note on the invite thing, all the more reason to swap it.

Reply:
I'm going to leave as is for now. I did verify that our other groups DO work with the current setting for sending invites, and when you open it up to everyone - all the details are present (person's email).